src/components/selector-dropdown/SelectorDropdown.scss

```diff
@@ -13,8 +13,9 @@
         @include bdl-Overlay-container;
     }
 
-    .overlay-wrapper,
+    .SelectorDropdown-overlay,
     .overlay {
+        position: relative;
```

Comment:
what's this for? is something using `position: absolute;`?

Reply:
The overlay css adds position: absolute to the parent of the PopperComponent which breaks the react-popper usage so I'm defining it here.

Reply:
I moved this to `PillSelectorDropdown` as it seems to be the only place we need `position: relative;`. 

@ivanthai please feel free to make changes if that's not the case.